File: BankKycSystemdotnet/BankKycSystem/DTOs/KycUploadRequestDTO.cs
using BankKycSystem.Enums;

namespace BankKycSystem.DTOs
{
    public class KycUploadRequestDTO
    {
        public string PoiType { get; set; } 
        public string PoaType { get; set; } 
        public IFormFile PoiDocument { get; set; } 
        public IFormFile PoaDocument { get; set; } 
        public AccountType AccountType { get; set; } 
    }
}

File: bank-kyc-system/src/main/java/com/bank/project/dto/KycUploadRequestDTO.java
package com.bank.project.dto;

import com.bank.project.enums.AccountType;
import jakarta.validation.constraints.NotNull;
import lombok.*;
import org.springframework.web.multipart.MultipartFile;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class KycUploadRequestDTO {

    @NotNull(message = "POI document is required")
    private MultipartFile poiDocument;  // 

    @NotNull(message = "POA document is required")
    private MultipartFile poaDocument;  // 

    @NotNull(message = "Account type is required")
    private AccountType accountType; // 
}
